### DISCUSSION TITLE
Debug in Xcode

### DISCUSSION THREAD

**Discussion by MingHieu** [RocketChat/Rocket.Chat.ReactNative#5035] [Q&A] [answered]
- On Android, I can read native log with Android Studio from Logcat section.
- On IOS, how can I achieve the same thing ? I can't see any log from debug console.
![Ảnh màn hình 2023-04-17 lúc 10 07 17](https://user-images.githubusercontent.com/77161145/232368470-30c622ee-5380-489e-879d-4f70d2f19ac6.png)

**Comment by dnlsilva** [comment]
What kind of log are you wanting to see?
`console.log`?

**Reply by MingHieu** [reply]
I want log from NSLog of Objective-C or print of Swift

**Comment by MingHieu** [comment, marked as answer]
Finally found this! Just untick OS_ACTIVITY_MODE
![Ảnh màn hình 2023-05-17 lúc 14 54 04](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/77161145/907cffd9-fefa-4b9f-94b1-f6f240bb6fcd)
